styles/util/index.scss

```diff
@@ -3,3 +3,11 @@
 @use 'colors';
 @use 'focus';
 @use 'layout';
+
+@tailwind utilities;
+
+@layer utilities {
+  .hyp-u-border {
+    @apply border border-solid border-grey-3;
+  }
```

Comment:
This is an example of an atomic pattern, which can get confused with true utility classes. Here it is implemented by composing tailwind utilities.